docs/sphinx/user_guide/feature/sort.rst

```diff
@@ -0,0 +1,159 @@
+
+A few important notes:
+
+.. note:: * All RAJA sort operations are in the namespace ``RAJA``.
+          * Each RAJA sort operation is a template on an *execution policy*
+            parameter. The same policy types used for ``RAJA::forall`` methods
+            may be used for RAJA sorts.
+          * RAJA sort operations accept an optional *comparator* argument so
+            users can perform different types of sort operations. If
+            no operator is given, the default is a 'less' operation and
+            the result is **non-decreasing**.
+
+Also:
+
+.. note:: * For sorts using the CUDA back-end, RAJA uses the implementations
+            provided by the NVIDIA cub library. For information please see
+            :ref:`build-external-tpl-label`.
+          * The RAJA CUDA back-end implementation only supports sorting
+            arithmetic types using RAJA operators less and greater.
+
+Please see the :ref:`sort-label` tutorial section for usage examples of RAJA
+sort operations.
+
+-----------------
+Sort Operations
+-----------------
+
+In general, a sort operation takes a sequence of numbers 'x' and a binary
+comparison operator 'op' that forms a strict weak ordering of elements in input
+sequence x and produces a sequence of numbers 'y' as output. The output sequence
+is a permutation of the input sequence where each pair of elements 'a' and 'b,'
+where a is before b in the output sequence, satisfies ~(b op a).
+Sorts are stable if they always preserve the order of equivalent elements,
+where equivalent elements satisfy ~(a op b) and !(b op a).
+
+A **stable sort** takes an input sequence where elements labeled a\ :sub:`i`
+for all i are equivalent elements and a\ :sub:`i` appears before a\ :sub:`j` if
+i < j
+
+   x = { a\ :sub:`0`\, b\ :sub:`0`\, a\ :sub:`1`\, ... }
+
+and calculates the sorted output sequence that preserves the order of equivalent
+elements, so a\ :sub:`i` still appears before a\ :sub:`j` if i < j:
+
+   y = { a\ :sub:`0`\, a\ :sub:`1`\, b\ :sub:`0`\, ... }
+
+An **unstable sort** may not preserve the order of equivalent elements and
+may produce either of the following output sequences.
+
+   y = { a\ :sub:`0`\, a\ :sub:`1`\, b\ :sub:`0`\, ... }
+
+   or
+
+   y = { a\ :sub:`1`\, a\ :sub:`0`\, b\ :sub:`0`\, ... }
+
+---------------------
+RAJA Unstable Sorts
+---------------------
+
+RAJA unstable sort operations look like the following:
+
+ * ``RAJA::sort< exec_policy >(container)``
+ * ``RAJA::sort< exec_policy >(container, comparator)``
+ * ``RAJA::sort< exec_policy >(iter, iter + N)``
+ * ``RAJA::sort< exec_policy >(iter, iter + N, comparator)``
+
+For example sorting the 'in' array filled with this sequence of values::
+
+   6 7 2 1 0 9 4 8 5 3 4 9 6 3 7 0 1 8 2 5
+
+by performing a sequential unstable sort operation using the following code::
+
+.. literalinclude:: ../../../examples/tut_sort.cpp
+   :start-after: _sort_seq_start
+   :end-before: _sort_seq_end
+   :language: C++
```

Comment:
I've tried adding an example here but the include isn't working like it does in the tutorial.
Does anyone know what's going on here?

Reply:
Are you looking at the generated docs on your branch on RTD?

Reply:
Your syntax looks correct.

Reply:
Do you need one more '../' in the path?

Reply:
giving it a shot

Reply:
If it still doesn't work, let me know. I can try to debug.

Reply:
Adding the extra ../ didn't fix it.

Reply:
I'll try to make time to look at it tomorrow morning.